music_album.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'item'
+require_relative 'genre'
+require 'date'
+
+class MusicAlbum < Item
+  attr_accessor :name, :genres, :publish_date, :on_spotify
+  attr_writer :genre
+
+  def initialize(name, publish_date, on_spotify: false, genres: [])
+    super()
+    @name = name
+    @publish_date = Date.parse(publish_date)
+    @on_spotify = on_spotify
+    @genres = genres
+  end
+
+  def can_be_archived?
+    (Date.today.year - @publish_date.year) > 10 && @on_spotify
+  end
```

Comment:
- [ ] Please `can_be_archived` should override the method from the parent class, as I can see you are just getting the code from the parent class instead of using the `super` keyword to do this job for you.